packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -257,9 +285,62 @@ export default class AiAssistantPanel extends Component<Signature> {
         --icon-color: var(--boxel-light);
         margin-left: var(--boxel-sp-xs);
       }
+
+      .past-sessions-button-active::before {
+        content: '';
+        position: absolute;
+        top: -105px;
+        left: -55px;
+        width: 250px;
+        height: 250px;
+        background: conic-gradient(
+          #ffcc8f 0deg,
+          #ff3966 45deg,
+          #ff309e 90deg,
+          #aa1dc9 135deg,
+          #d7fad6 180deg,
+          #5fdfea 225deg,
+          #3d83f2 270deg,
+          #5145e8 315deg,
+          #ffcc8f 360deg
+        );
+        z-index: -1;
+        animation: spin 4s infinite linear;
+      }
+
+      .past-sessions-button-active::after {
+        content: '';
+        position: absolute;
+        top: 1px;
+        left: 1px;
+        right: 1px;
+        bottom: 1px;
+        background: #272330;
+        border-radius: inherit;
+        z-index: -1;
+      }
+
+      .past-sessions-button-active {
+        position: relative;
+        display: inline-block;
+        border-radius: 3rem;
+        color: white;
+        background: #272330;
```

Comment:
css-variable for this color is `--boxel-700`

Reply:
Cool, I adjusted it